pkg/features/kube_features.go

```diff
@@ -203,10 +203,10 @@ const (
 	BlockVolume utilfeature.Feature = "BlockVolume"
 
 	// owner: @pospispa
-	//
 	// alpha: v1.9
-	// Postpone deletion of a persistent volume claim in case it is used by a pod
-	PVCProtection utilfeature.Feature = "PVCProtection"
+	//
+	// Postpone deletion of a PV or a PVC when they are being used
+	StorageProtection utilfeature.Feature = "StorageProtection"
```

Comment:
This makes it sound like something else entirely.  Can we rename this to "PVDeleteInhibit" or "VolumeInUseProtection" or something?

Reply:
we did not pay much attention on the name. in order to reuse this feature gate for PV(it is introduced for PVC), maybe later also for SC, we change it to StorageProtection . 
"PVDeleteInhibit" or "VolumeInUseProtection" sounds like just for PV ?

Reply:
I opened https://github.com/kubernetes/kubernetes/issues/59639